base/client.jl

```diff
@@ -57,10 +75,13 @@ function repl_color(key, default)
     haskey(text_colors, c_conv) ? c_conv : default
 end
 
-warn_color()   = repl_color("JULIA_WARN_COLOR", default_color_warn)
-info_color()   = repl_color("JULIA_INFO_COLOR", default_color_info)
-input_color()  = text_colors[repl_color("JULIA_INPUT_COLOR", default_color_input)]
-answer_color() = text_colors[repl_color("JULIA_ANSWER_COLOR", default_color_answer)]
+error_color() = repl_color("JULIA_ERROR_COLOR", default_color_error)
+warn_color()  = repl_color("JULIA_WARN_COLOR" , default_color_warn)
+info_color()  = repl_color("JULIA_INFO_COLOR" , default_color_info)
+
+# Print input and answer in bold.
+input_color()  = text_colors[:bold] * text_colors[repl_color("JULIA_INPUT_COLOR", default_color_input)]
+answer_color() = text_colors[:bold] * text_colors[repl_color("JULIA_ANSWER_COLOR", default_color_answer)]
```

Comment:
Perhaps. On the other hand it makes sense to have all the ENV color stuff in the same place.